hkm/migrations/0040_auto_20200903_0708.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2020-09-03 04:08
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Could you change the filename so that it's more descriptive? It looks like the migration is changing only the name of the `albums` field so maybe something like `0040_update_showcase_albums_description.py`?

Once it's changed, modify also the dependency in the other file.

Reply:
File is now renamed.